src/nvim/decoration.h

```diff
@@ -60,7 +60,9 @@ struct Decoration {
   int col;  // fixed col value, like win_col
   int virt_text_width;  // width of virt_text
   char *sign_text;
+  char *sign_name;
```

Comment:
Can we add this to `ExtmarkInfo`? It's helpful for `'statuscolumn'` consumers to know the name attached to a sign mark. So I think `nvim_buf_get_extmarks()` should return the sign name for legacy signs but perhaps we even want to make this a public extmark property?

Reply:
`ExtmarkInfo` is going a way anyway. for now can just access it via the `Decoration *` field, no?

Reply:
Yeah adding to `ExtmarkInfo` implies implies adding it to to the `nvim_buf_get_exmarks()` info array, which is what I meant. I can skip `ExtmarkInfo` sure.